src/scripts/main.js

```diff
@@ -5,4 +5,65 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+for (const person of people) {
+  const row = document.createElement('tr');
+
+  for (const key in person) {
+    switch (key) {
+      case 'name':
+        createCell(person, 'name', row);
+        break;
+      case 'sex':
+        createCellGender(person, row);
+        break;
+      case 'born':
+        createCell(person, 'born', row);
+        break;
+      case 'died':
+        createCell(person, 'died', row);
+        break;
+    }
+  }
+  createCellAge(person, row);
+  createCellCentury(person, row);
+  table.append(row);
+}
+
+function createCell(object, key, row) {
+  const cell = document.createElement('td');
+  const cellText = document.createTextNode(object[key]);
+
+  cell.appendChild(cellText);
+  row.appendChild(cell);
+}
+
+function createCellGender(object, row) {
+  const cell = document.createElement('td');
+  let cellText = '';
+
+  if (object.sex === 'f') {
+    cellText = document.createTextNode('Female');
+  } else {
+    cellText = document.createTextNode('Male');
+  }
```

Comment:
На такі речі непогано би підійшов тернарний оператор
```suggestion
  const cellText = object.sex === 'f' 
    ? document.createTextNode('Female')
    : document.createTextNode('Male');
```